src/_ctf-private/ctf_sign-up-banner/CtfSignUpBanner.tsx

```diff
@@ -0,0 +1,93 @@
+import { Theme, Typography } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import clsx from 'clsx';
+
+import Logo from './CtfLogo.svg';
+
+import { Link } from '@src/components/shared/link';
+
+const signUpLink =
+  'https://www.contentful.com/starter-templates/marketing-website/sign-up/?action=create_starter_template&template_name=marketing';
+const systemUIFontFamilies =
+  '-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"';
```

Comment:
I reckon from the design & the marketing website, the font family is

`"Avenir Next W01", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"`